pjsip-apps/src/samples/stateful_proxy.c

```diff
@@ -346,6 +346,7 @@ static pj_bool_t proxy_on_rx_response( pjsip_rx_data *rdata )
     status = pjsip_endpt_send_response(global.endpt, &res_addr, tdata,
 				       NULL, NULL);
     if (status != PJ_SUCCESS) {
+	pjsip_tx_data_dec_ref(tdata);
```

Comment:
After searching for `pjsip_endpt_create_response` in the library code, looks like there are some other places need to be fixed as well.

We can fix those if you like, but, in case it has not been set, we will need you to allow us to apply changes to the fork as described [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

Reply:
Yes, please those. Thank you! ("allow edits by maintainers" is selected)

Reply:
@nanangizz - I made changes you suggested regarding docu and added `pjsip_tx_data_dec_ref(tdata);` to `pjsip_endpt_send_response` in case of an error. Please check and feel free to improve/update/fix.